sdk/communication/AzureCommunicationChat/Source/Custom/AzureCommuncationChatClient+Convenience.swift

```diff
@@ -161,10 +161,10 @@ extension AzureCommunicationChatClient {
     ///     success.
     public func listChatThreadMembers(
```

Comment:
`listChatParticipants`.

This layer was created (rather hastily) so that customers don't have to say `client.azureCommandChatService.blah` and can just say `client.blah` the way they could with the first release. Since you are already writing a convenience layer, feel free to retain or remove as much of this as you like.

Reply:
Ok cool! I moved it out of custom folder & updated some of the names commented above with the API renaming, kept as is otherwise & will leave any changes for the refactor PR.